controllers/utils/etcd.go

```diff
@@ -0,0 +1,38 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+
+	v1 "k8s.io/api/policy/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const etcdNamespace = "openshift-etcd"
+
+// +kubebuilder:rbac:groups="policy",resources=poddisruptionbudgets,verbs=get;list;watch
+
+// IsEtcdDisruptionAllowed checks if etcd disruption is allowed
+func IsEtcdDisruptionAllowed(ctx context.Context, cl client.Client, log logr.Logger) (bool, error) {
+	log = log.WithName("etcd-pdb-checker")
+	pdbList := &v1.PodDisruptionBudgetList{}
+	if err := cl.List(ctx, pdbList, &client.ListOptions{Namespace: etcdNamespace}); err != nil {
+		return false, err
```

Comment:
NIT: Can we log the error in case it occurs

Reply:
I have looked only in https://github.com/medik8s/node-healthcheck-operator/pull/272/files#diff-42b14dc2a70877933db4647434fcbe9b65a8859c7ec1911b0c9da7361c8e98b1R753 and not in https://github.com/medik8s/node-healthcheck-operator/pull/272/files#diff-42b14dc2a70877933db4647434fcbe9b65a8859c7ec1911b0c9da7361c8e98b1R574 and etc.